include/roboteam_ai/bt/BehaviorTree.hpp

```diff
@@ -9,11 +9,13 @@ namespace bt {
 class BehaviorTree : public Node {
         FRIEND_TEST(BehaviorTreeTest, it_sets_blackboards);
     public:
-
+        std::string name;
         using Ptr = std::shared_ptr<BehaviorTree>;
 
         BehaviorTree();
 
+        BehaviorTree(std::string name);
```

Comment:
A constructor is a public function so it needs documentation. I don't know if the name string should be a foreign key for example.